backend/server/src/db/model/drv/queries.rs

```diff
@@ -80,6 +80,49 @@ pub async fn insert_drv_graph(
     Ok(())
 }
 
+/// This will insert a vec of <drv, referrence> into
+/// the database. This only creates the reference+referrer relationship
+/// Drvs are assumed to already be populated
```

Comment:
```suggestion
/// This will insert a slice of drvs and <reference, referrer> into
/// the database. The reference relationships assume that the drvs
/// have been inserted previously, or passed in this call as well
```